forms.php

```diff
@@ -0,0 +1,84 @@
+<?php 
+    $valid=array();
+    $error=array();
+    if ($_POST) {
+        $title=$_POST['title'];
+        if (!$title)
+        {
+            $errors['title']="Името е задължително поле";
+        }
+        elseif(strlen('title')>150)
+        {
+            $errors['title']="Името трябва да бъде не повече от 150 символа";
+        }
+        else
+        {
+            $valid['title']=$title;
+        }
+        
+        $name=$_POST['name'];
+        if (!$name)
+        {
+            $errors['name']='Името е задължително поле';
+        }
+        elseif(strlen($name)>200)
+        {
+            $errors['name']="Името трябва да бъде не повече от 200 символа";
+
+        }
+        else
+        {
+            $valid['name']=$name;
+        }
+
+        $description=$_POST['description'];
+        if (!$description)
+        {
+            $errors['description']='Описанието е задължително поле';
+        }
+        elseif(strlen($description)<10)
```

Comment:
Please extract all constant values into constants/variables and give them meaningful names. This way you will know what is the purpose of the value when reading the code. For example here you would be able to reuse the value of 10 in the error message too here that way.